ocaml/xapi/create_misc.ml

```diff
@@ -411,6 +434,8 @@ let make_software_version ~__context =
 		"xencenter_min", Xapi_globs.xencenter_min_verstring;
 		"xencenter_max", Xapi_globs.xencenter_max_verstring;
 		"network_backend", Network_interface.string_of_kind (Net.Bridge.get_kind dbg ());
+		"xen_livepatch_list", get_xen_livepatch_list ();
```

Comment:
The design doc states that the key name should be \<component\>-livepatches, so I think it should be xen-livepatches and kernel-livepatches.